src/qz/installer/Installer.java

```diff
@@ -170,6 +170,31 @@ public Installer removeLibs() {
         return this;
     }
 
+    public Installer cleanupLegacyLogs(int rolloverCount) {
+        // Convert old < 2.2.3 log file format
+        Path logLocation = USER_DIR;
+        int nextFile = 0;
+        for(int i = 1; i <= rolloverCount; i++) {
+            // Old: debug.log.1
+            File oldFile = logLocation.resolve("debug.log." + i).toFile();
+            if(oldFile.exists()) {
```

Comment:
why do we specify a rolloverCount instead of just looping until the first non-existent .log.n file?

Reply:
The original intention was to reduce the scope.  At that time I didn't know that the log engine was smart enough to cleanup similarly named files, so I only wanted to touch what we created.  Knowing what we know now, this can safely increment as long as there's files. I'll  do that now.

Reply:
Should be addressed via 5585d84.